test/test_experimental.py

```diff
@@ -171,20 +157,8 @@ def test_eri():
     # H2 molecule in sto-3g: See equation 3.235 of Szabo and Ostlund
     h2 = molecule("h2")
     basis = basisset(h2, "sto-3g")
-    indices = [(0, 0, 0, 0), (0, 0, 1, 1), (1, 0, 0, 0), (1, 0, 1, 0)]
-    expected = [0.7746, 0.5697, 0.4441, 0.2970]
-
-    for ijkl, expect in zip(indices, expected):
-        actual = eri_orbitals([basis.orbitals[aoid] for aoid in ijkl])
-        assert_allclose(actual, expect, atol=1e-4)
-
-
-def test_eri_basis():
-    # H2 molecule in sto-3g: See equation 3.235 of Szabo and Ostlund
-    h2 = molecule("h2")
-    basis = basisset(h2, "sto-3g")
-
```

Comment:
These deletions seem a bit orthogonal, and will contribute to test time?

Reply:
Good point, I think I'd like to reorganise the tests to break apart the growing file `test_experimental.py`

I started doing this on another branch but I think it would be cleaner to do this kind of reorg separately from other source level changes.  I can revert these changes and open a new issue to organise the tests (including removing this duplicated test)

Does that seem reasonable?